jdk_11_maven/cs/rest-gui/market/market-rest/pom.xml

```diff
@@ -27,6 +27,7 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+				<version>1.4.4.RELEASE</version>
```

Comment:
hmmm... are those `1.4.4.RELEASE` the same version of SpringBoot used in those SUTs?

Reply:
you are right. we should use the same one as SUT. now they are updated and checked with mvn on the server.